src/base/system_parameter.c

```diff
@@ -1450,6 +1454,16 @@ const char *PRM_HA_PING_HOSTS = "";
 static const char *prm_ha_ping_hosts_default = NULL;
 static unsigned int prm_ha_ping_hosts_flag = 0;
 
+const char *PRM_HA_TCP_PING_HOSTS = "";
+static const char *prm_ha_tcp_ping_hosts_default = NULL;
+static unsigned int prm_ha_tcp_ping_hosts_flag = 0;
+
+int PRM_HA_PING_TIMEOUT = 1000;
```

Comment:
Nit.
Other timeout-related parameters are set in seconds. 
To avoid confusion, I think it may be helpful to add a suffix like `IN_MSEC` to the parameter name to indicate that it is in milliseconds.

Reply:
In the manual, it is a little bit inconsistent about you commented on as following: (see the lock_timeout and _seconds suffix)

- lock_timeout (msec)
- thread_connection_timeout_seconds (sec)
- thread_worker_timeout_seconds (sec)
- session_state_timeout (sec) 
- checkpoint_sleep_msecs (msec)
- group_commit_interval_in_msecs (msec)

But, I agree with your comments and I will apply it. And I want to change the default value from 1000 msec to 2000 msecs for tcp connection. Do you have any opinion about this?

Reply:
I thought the reason that default value of this parameter is set to 1 second was because the timeout of the existing ping behavior was hard-coded to 1 second (PING_COMMAND_FORMAT : `"ping -w 1 -c 1 %s >/dev/null 2>&1; "`). However, if there is no need to maintain this, wouldn't it be better to change it to 5 seconds? I mean how about following the PRM_ID_TCP_CONNECTION_TIMEOUT?

Reply:
@joohok The ha_ping_timeout will be used for the ping command some day. But now, it is good idea to me to follow the connection_timeout for the default value of tcp ping time.

Reply:
When I check the system parameters which is related to the connection time out (connection_timeout, thread_connection_timeout_seconds, etc) It would be better to set ping_timeout in sec units rather than msecs. In this case I could think of ha_ping_timeout_seconds but I prefer not to append suffix. please comment on.

Reply:
Agreed. 
I agree to use seconds as the unit of parameter, and I think there's no need to add the suffix to the parameter name.